jest.config.json

```diff
@@ -1,11 +1,6 @@
 {
-  "roots": [
-    "test"
-  ],
-  "testPathIgnorePatterns": [
-    "/node_modules/",
-    "<rootDir>/test/main/signers/ledger/adapter"    
-  ],
+  "roots": ["test"],
```

Comment:
wyt about excluding these types of files (perhaps `.config.json`) or some such? I kind of like the readability of the previous style, especially for configuration files where compacting things into one line isn't really important

Reply:
I'm honestly more inclined to let Prettier do its thing everywhere - managing exemptions would be more work